templates/secrets.yaml

```diff
@@ -258,6 +259,17 @@ stringData:
         max_idle_conns: {{ $maxIdleConns }}
         conn_max_lifetime: {{ $conMaxLifeTime }}
 
+    # Configuration for the Signing Key Server (for end-to-end encryption).
+    signing_key_server:
```

Comment:
Wondering why you are getting an error, the `signing_key_server` was merged with the `federationapi` (https://github.com/matrix-org/dendrite/commit/ec716793eb86e7723ecfaa356e148334bc3921eb)

Reply:
probably because I was using and old build (I used the latest on Docker Hub, then realize it was 2 months ago). Removed those changes related to signing_key_server.